activerecord/lib/active_record/railties/databases.rake

```diff
@@ -25,7 +25,7 @@ db_namespace = namespace :db do
   end
 
   desc "Creates the database from DATABASE_URL or config/database.yml for the current RAILS_ENV (use db:create:all to create all databases in the config). Without RAILS_ENV or when RAILS_ENV is development, it defaults to creating the development and test databases."
-  task create: [:load_config] do
+  task create: [:environment, :load_config] do
```

Comment:
I think we need to change `load_config` to have `environment` as dependency. the `db:create:all` task now is not loading `environment` for example.

Reply:
Thanks for your comment. I agree that adding a dependency of `environment` to `load_config` is a better approach. Updated.